pallets/afloat/src/lib.rs

```diff
@@ -159,6 +200,16 @@ pub mod pallet {
 	{
 			ensure_signed(origin.clone())?;
 			let asset_id: T::AssetId = Default::default();
+			let min_balance: T::Balance = 1u32.into();
+
+			// Create asset
+			pallet_mapped_assets::Pallet::<T>::create(
```

Comment:
add a boolean in initial setup related the creation of the MA asset, something as 
```rust
pub fn initial_setup(
			origin: OriginFor<T>,
			creator: T::AccountId,
			admin: T::AccountId,
            create_asset: bool,
		) 
```
Just to make sure we can reuse an asset even if we delete all the storage maps